lib/cn/kblock_builder.c

```diff
@@ -1290,6 +1289,21 @@ kbb_set_merge_stats(struct kblock_builder *bld, struct cn_merge_stats *stats)
     bld->mstats = stats;
 }
 
+bool
+kbb_is_empty(struct kblock_builder *bld)
+{
+    return kblock_is_empty(&bld->curr);
+}
+
+void
+kbb_curr_kblk_minmax_keys(
+    struct kblock_builder *bld,
+    struct key_obj        **min_kobj,
+    struct key_obj        **max_kobj)
+{
+    wbb_min_max_keys(bld->curr.wbtree, min_kobj, max_kobj);
```

Comment:
I figured that wbb_min_max_keys() returns correct min/max only for a finalized wbtree.

@gsramdasi educated me that this is due to the staging area that wbb has for common prefix elimination.

I've now modified wbb_min_max_keys() to work correctly both for a finalized and a non-finalized wbtree.. Testing in progress...